Makefile

```diff
@@ -30,6 +30,7 @@ EMBEDASSETS ?= embedassets
 # Build flags
 VPREFIX := github.com/grafana/pyroscope/pkg/util/build
 GO_LDFLAGS   := -X $(VPREFIX).Branch=$(GIT_BRANCH) -X $(VPREFIX).Version=$(IMAGE_TAG) -X $(VPREFIX).Revision=$(GIT_REVISION) -X $(VPREFIX).BuildDate=$(GIT_LAST_COMMIT_DATE)
+GO_GCFLAGS   := all="-N -l"
```

Comment:
should we add ad DEBUG preffix or suffix here ?

Reply:
Good idea, added in https://github.com/grafana/pyroscope/pull/3517/commits/8d1035ab702ad26a754b45863a736a00456973e8